lib/extensions.js

```diff
@@ -0,0 +1,75 @@
+// Imports
+const fs = require('fs')
+const path = require('path')
+const foundationExtensions = require('../app/config').foundationExtensions || ['govuk-frontend']
```

Comment:
not sure `foundation` is the clearest word, is `priority` better?

Reply:
It would be good to bounce a few options around.  These are the lowest priority, they're the building blocks.